<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leaderboards</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="leader_label">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>20</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Leaderboards</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="leader_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>113</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>18</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Player</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Score</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
